### ISSUE TITLE
ReferenceError: match is not defined

### ISSUE THREAD

**Issue by SamMorgan** [stevenschobert/instafeed.js#692]
Appears to retrieve the data but fails to render due to this error

**Comment by stevenschobert** [comment]
Hey @SamMorgan, thanks for the report. Can you put together a runnable example of this bug so I can take a look?

Thanks!

**Comment by SamMorgan** [comment]
Fixed the issue, seems to be an issue with the script in the repository. I making a simplified copy and used the CDN script used on your example on codepen (easier since didn't know the NPM target) and it worked. Switched to the script to whats used on codepen on the full site and that works too. Making an example pushed me in the right direction, thanks!

**Comment by mylastore** [comment]
same here, I try changing the target and get the same error.

```
ReferenceError: assignment to undeclared variable match
    extractTags instafeed.js:238
    getItemData instafeed.js:269
    processData instafeed.js:178
    onResponseReceived instafeed.js:133
    callbackOnce instafeed.js:394
    apiRequestOnLoad instafeed.js:435
```

**Comment by stevenschobert** [comment]
Hey everyone! Just wanted to let y'all know I tracked down the source of this error, and it will be fixed in the next release.

**Comment by stevenschobert** [comment]
This bug should be fixed in [v2.0.0-rc3](https://github.com/stevenschobert/instafeed.js/releases/tag/v2.0.0-rc3). Sorry it took me so long to get to!
